packages/applications-service-api/src/utils/documentFilterLabelMapper.js

```diff
@@ -0,0 +1,24 @@
+const mapDocumentFilterLabel = (filterName, filterValue) => {
+	try {
+		return {
+			stage: {
+				1: 'Pre-application',
+				2: 'Acceptance',
+				3: 'Pre-examination',
+				4: 'Examination',
+				5: 'Recommendation',
+				6: 'Decision',
+				7: 'Decided'
```

Comment:
I think that may be Post-decision looking at the design - 
https://applications-prototype.up.railway.app/design-sprint-6/wide-nav/doc-library?filtersapplied=true